index.html

```diff
@@ -2,261 +2,343 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./styles.css">
-    <!-- <link rel="stylesheet" href="./font/stylesheet.css"> -->
-    <title>Portfolio</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;500;600;700&display=swap" rel="stylesheet">
+  <title>Portfolio</title>
 </head>
 
 <body>
-    <div class="container">
-        <header class="elements">
-            <a href="https://github.com/calebchris000" class="item logo">My logo</a>
-            <nav class="item harmburger"><img src="./Graphics/Normal Button/Tertiary/Icons/Union.svg" alt=""></nav>
-
-        </header>
-        <section>
-            <h1 class="greeting">
-                I’m Caleb<br>
-                Glad to see you!
-            </h1>
-
-
-            <p class="content">
-                I’m a software developer!
-                I can help you build a product ,
-                feature or website Look through
-                some of my work and experience! If
```

Comment:
as the previous reviewer suggested, Kindly adjust the padding of this section to match the original design

![image](https://user-images.githubusercontent.com/98453979/213493414-3e921589-132d-4625-8589-81bc5a9312f1.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/213493799-14fdee4f-1c11-4cf1-a766-a672a349001f.png)